angular15-vue3/shell/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const webpack = require('webpack');
+const ModuleFederationPlugin = require('webpack/lib/container/ModuleFederationPlugin');
+
+module.exports = {
+  output: {
+    publicPath: 'http://localhost:4200/',
+    uniqueName: 'shell',
+    scriptType: 'text/javascript',
+  },
+  optimization: {
+    runtimeChunk: false,
+  },
+  plugins: [
+    new ModuleFederationPlugin({
+      remotes: {
+        profile: 'profile@http://localhost:4201/remoteEntry.js}',
+      },
```

Comment:
@belinef redundant character at the end of the profile url

Reply:
resolved